src/index.html

```diff
@@ -1,13 +1,54 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/style.css">
+  </head>
+  <body>
+    <h1>Product cards</h1>
+    <div class="product" data-qa="card">
+      <div>
+        <img 
+          src="images/imac.jpeg" 
+          alt="iMac Photo" 
+          class="product__img"
+        >
+      </div>
+      <div class="product__description">
```

Comment:
See checklist about empty lines.
```suggestion
      </div>
      
      <div class="product__description">
```